extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1936,16 +1936,19 @@ describes.realWin('amp-analytics', {
       };
     };
 
-    const runResourceTimingTest = function(entries, config, expectedPing) {
-      sandbox.stub(win.performance, 'getEntriesByType').returns([entries]);
+    this.timeout(400);
+
+    const runResourceTimingTest = function(
+      entries, config, expectedPing, timeout = 100) {
+      sandbox.stub(win.performance, 'getEntriesByType').returns(entries);
       const analytics = getAnalyticsTag(config);
-      return waitForSendRequest(analytics).then(() => {
+      return waitForSendRequest(analytics, timeout, 2).then(() => {
```

Comment:
Why always expect 2 requests? I see we do have some test only wait for one request. We can make `expectedPing` to be an array, and configure the test based on the array's length.   WDYT